cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/Transformer.java

```diff
@@ -26,6 +27,8 @@
 
     String keyToTransform();
 
+    void setSensitivityLevel(SensitivityLevel sensitivityLevel);
```

Comment:
At compiler/modelling level you added overloaded methods (stateless), but at transformer level you added a flag / setter method (stateful) - I don't think you should mix these behaviors

IMHO opinion you shoul overload the transformer too:
```
    TransformModellingResult<T> transform(F rawData);
    TransformModellingResult<T> transform(F rawData, sensitivityLevel);

    delete this method: --------------> void setSensitivityLevel(SensitivityLevel sensitivityLevel);
```